src/Service/VendorService/UploadService/UploadServiceVendorService.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * @file
+ * Service that don't do anything. It exists to enable an vendor for the upload service.
```

Comment:
don't  =>   does not

an vendor   =>   a vendor

Reply:
Fixed